src/main/kotlin/day17/mapper/PhotoMapper.kt

```
@@ -0,0 +1,27 @@
+    fun mapToModel(dto: PhotoDto): Photo {
+        return Photo(
+            id = dto.id ?: 0, // id가 null이거나 숫자가 아니면 기본값 0
+            type = PhotoType.values().find { it.name.equals(dto.type, ignoreCase = true) } ?: PhotoType.UNKNOWN,
+            title = dto.title,
+            content = dto.content,
+            url = dto.url,
+            caption = dto.caption,
+            createdAt = dto.createdAt?.let {
+                try {
+                    LocalDate.parse(it, dateFormatter)
+                } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)